cassandane/Cassandane/Cyrus/Conversations.pm

```diff
@@ -213,6 +213,10 @@ sub test_move_200
     my ($self) = @_;
     my %exp;
 
+    xlog $self, "Enable compactids";
```

Comment:
This feels like what we really want is a `:CompactIds` magic, though I see later that we're going to run into issues with it being for multiple users... should we have a config option that always turns them on for new users instead in the Cass tests?